### PR TITLE
Fixed buttons, icon tab indexes and created new component colors

### LINKED ISSUE #110: [BUG] Fix buttonless buttons
Many of the buttons in the UI are less than optimal. As an example, the action buttons in the top bar of each sidebar are just SVG icons with a `click` listener. These are impossible to use without a mouse. You cannot tabulate to focus them, and you cannot press enter to open them.

Things to check for all buttons in the UI:
- it is a real \<button\>. anything else comes with 10000 pitfalls that you haven't thought about. don't do it.
- when focusing the button, it will show a glowing outline to make it clear that it is in focus
- you can tabulate to and from the button
- when hovering over the button, the mouse cursor changes to show that the button can be clicked
- when hovering over the button, the button is highlighted to show that it can be clicked
- when clicking the button, it changes appearance slightly to make it clear that the button is being pressed
- the button has an accesible name. e.g. a button containing an SVG must have a text-based label for screen readers

If you are in doubt about anything, you can probably find the answer here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG
Keyboard stuff here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard
Text label stuff here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#interactive_elements_must_be_labeled

I would recommend using the "Inspect Accessibility Properties" in Firefox devtools, as it helps you find these issues. For example, here it found a button that cannot be used via keyboard (tabulate, press enter), and which also does not have a screen reader label
image

You can hold shift and right click any element in the UI, then click "Inspect Accessibility Properties" to get to this menu :)

### PR DESCRIPTION
**Pull request by BaBrixx**
Closes #110

### Changes/Additions:
- Changed the tab indexes of icons to not make them selectable when pressing tab.
- Wrapped clickable icons in buttons.
- Changed the colors of components.

### CODE REVIEW

File: src/lib/components/query/QueryDropDownMenu.svelte [hunk 1]
```diff
@@ -17,30 +17,26 @@
 	const menuId = `query-menu-${index}`;
 	let button: HTMLElement;
 
+	/**
+	 * Function for toggling the isPeriodic variable
+	 * @param {MouseEvent} event
```
Comment (AlbertHald) [inline src/lib/components/query/QueryDropDownMenu.svelte:22]: This is bing, thank you for coming to my TED talk

**Review by AlbertHald — DISMISSED** [review]
If you know, then you know, but if you don't know then now you know that you didn't know what you needed to know

**Review by Jesper276 — DISMISSED** [review]
LGTM

**Review by atjn — APPROVED** [review]
LGTM

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

**Pull request merged**